velox/common/base/Status.h

```diff
@@ -485,6 +485,29 @@ void Status::moveFrom(Status& s) {
     VELOX_RETURN_IF(!__s.ok(), __s);                          \
   } while (false)
 
+/// Return with given status wrapped in folly::Unexpected if condition is met.
```

Comment:
can we also add some examples in the documentation on the top of the file?

Reply:
@pedroerp Added the doc. Please help to review again. Thanks!